x-pack/test/security_solution_endpoint/apps/integrations/policy_details.ts

```diff
@@ -400,8 +400,18 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await (await testSubjects.find('policyDetailsBackLink')).click();
         await testSubjects.existOrFail('endpointIntegrationPolicyForm');
       });
-      it.skip('should not show host isolation exceptions card because no entries', async () => {
-        await testSubjects.missingOrFail('hostIsolationExceptions-fleet-integration-card');
+      it('should show host isolation exceptions card and link should go back to policy', async () => {
```

Comment:
while all of these tests (the whole file, not only these four tests) were being passed, another code change occurred:
- https://github.com/elastic/kibana/pull/145123

the PR above deleted the `useCanSeeHostIsolationExceptionsMenu()` hook, which were used on Policy Details page to hide the HIE card. sine this change, we show the HIE card with zero entries, just as the other cards